gossip/src/crds.rs

```diff
@@ -485,21 +485,30 @@ impl Crds {
             // all associated values.
             let origin = CrdsValueLabel::LegacyContactInfo(*pubkey);
             if let Some(origin) = self.table.get(&origin) {
-                if now < origin.local_timestamp.saturating_add(timeout) {
+                if origin
```

Comment:
Might not be bad to add a function to LegacyContactInfo to abstract away getting the recency timestamp. Should help with readability if nothing else

Reply:
This is actually `VersionedCrdsValue`.
I think the only place we do this `wallclock().min(entry.local_timestamp)` is inside this function, but we can revisit if more places need the same abstraction.